### PR TITLE
feat: add api_url option for Jira service

### PR DESCRIPTION
**Pull request by grv87**
Also mark `project_key` in docs as optional

### CODE REVIEW

File: gitlab/resource_gitlab_service_jira.go [hunk 1]
```diff
@@ -46,6 +46,12 @@ func resourceGitlabServiceJira() *schema.Resource {
 				Required:     true,
 				ValidateFunc: validateURLFunc,
 			},
+			"api_url": {
+				Type:         schema.TypeString,
+				Optional:     true,
```
Comment (roidelapluie) [inline gitlab/resource_gitlab_service_jira.go:51]: Should it also be computed? Wouldn't the underlying jira give it back anyway?

Do we have a test without the parameter?

**Comment by AlexGluck** [comment]
We interested this PR, because we maintain fork 1,5 year for this provider. I think if not set just skip this option for gitlab api requests. How can we help for merge this?

**Review by timofurrer — APPROVED** [review]
LGTM @nagyv (also please rebase and retrigger the checks).

**Comment by nagyv** [comment]
@grv87 Please, rebase to retrigger the checks. I would love to merge your PR afterwards.

**Comment by timofurrer** [comment]
I took the liberty to resolve the conflicts and apply some improvements / fixed.

Going to merge now ;)

File: internal/provider/resource_gitlab_service_jira.go [hunk 2]
```diff
@@ -182,9 +189,14 @@ func resourceGitlabServiceJiraRead(ctx context.Context, d *schema.ResourceData,
 		return diag.FromErr(err)
 	}
 
+	log.Printf("[TIMOOOO:] %+v", jiraService)
```
Comment (armsnyder) [inline internal/provider/resource_gitlab_service_jira.go:192]: @timofurrer lol, probably not intended?

Comment (timofurrer) [reply internal/provider/resource_gitlab_service_jira.go:192]: omg 😂  soo sorry. Please merge: https://github.com/gitlabhq/terraform-provider-gitlab/pull/880

**Pull request merged**